### PR TITLE
Prepare new release 4.8.0+build_125

### PR DESCRIPTION
**Pull request by Millefeuille42**
#6386

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 4.9.0 / TBA
+
+### ✨ New Features
+
+- New Logbook content: A logbook is a type of content that features logbook entries. It displays these entries chronologically as a timeline, with an emphasis on the time and the order.
+- Added real delete of content for workspace manager
```
Comment (Relf-Algoo) [inline CHANGELOG.md:8]: - Add possibility to really delete content, accessible to space managers.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+- Added transformation from news into thread
```
Comment (Relf-Algoo) [inline CHANGELOG.md:9]: - Add functionnality to transform news into thread

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 4.9.0 / TBA
+
+### ✨ New Features
+
+- New Logbook content: A logbook is a type of content that features logbook entries. It displays these entries chronologically as a timeline, with an emphasis on the time and the order.
```
Comment (Relf-Algoo) [inline CHANGELOG.md:7]: - New content: Logbook. A logbook is a type of content that features logbook entries. These entries are displayed as a timeline, with an emphasis on the time and the order, most recent at the top.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+- Added real delete of content for workspace manager
+- Added transformation from news into thread
+- The state of the sidebar is now saved in the browser
```
Comment (Relf-Algoo) [inline CHANGELOG.md:10]: - The cache of the browser used now saves the states of the navigation sidebar. Hidden sub-spaces will stay hidden.

**Comment by Skylsmoi** [comment]
Issue to implement these comments here: https://github.com/tracim/tracim/issues/6388

**Comment by Skylsmoi** [comment]
Also, we must add some information about the breaking change related to plugins with a link to https://github.com/tracim/tracim/issues/6385

**Pull request merged**